src/modules/85-cv/components/Notifications/components/NotificationRuleRow/NotificationRuleRow.tsx

```diff
@@ -153,6 +157,48 @@ const renderConnectedFields = (
           ) : null}
         </>
       )
+    case Condition.CODE_ERRORS:
+      return (
+        <>
+          <Layout.Vertical spacing="xsmall" padding={{ left: 'small', right: 'small' }}>
+            <Text>{getString('events')}</Text>
+            <FormInput.CustomRender
+              name={`conditions.${index}.eventStatus`}
+              render={() => {
+                return (
+                  <MultiSelectDropDown
+                    value={getArrayOrEmpty(eventStatus)}
+                    items={eventStatusOptions}
+                    className={css.field}
+                    onChange={option => {
+                      handleChangeField(notificationRule, option, 'eventStatus')
+                    }}
+                  />
+                )
+              }}
```

Comment:
Just personal preference, but I always prefer implicit returns for things like this.
```suggestion
              render={() => (
                <MultiSelectDropDown
                  value={getArrayOrEmpty(eventStatus)}
                  items={eventStatusOptions}
                  className={css.field}
                  onChange={option => {
                    handleChangeField(notificationRule, option, 'eventStatus')
                  }}
                />
              )}
```